README.md

```diff
@@ -15,7 +15,7 @@ Prerequisites:
 - Java SDK 1.8 (Build will not work on JDK 9 and above as of now)
 - Apache Maven
 
-Clone this repository, set the environment variable MVN\_HOME with the path to this repository's root directory and run `mvn install` in the root directory.
+Clone this repository, set the environment variable NMF\_HOME with the path to this repository's root directory and run `mvn install` in the root directory.
```

Comment:
How would it work in a standalone execution of applications from zipped NMF SDK snapshot?

Reply:
What exactly do you mean by zipped?